app/views/report/_role_list.html.haml

```diff
@@ -5,15 +5,7 @@
       .col-sm-5
         %h3
           = _("Reports")
-        #menu_roles_treebox.treeview-pf-hover.treeview-pf-select{:style => "overflow-y: hidden !important;"}
-      = render(:partial => "layouts/tree",
-        :locals         => {:tree_id => 'menu_roles_treebox',
-          :tree_name                 => 'menu_roles_tree',
-          :bs_tree                   => @menu_roles_tree,
-          :click_url                 => '/report/menu_editor/',
-          :onclick                   => 'miqMenuEditor',
-          :cookie_prefix             => "edit_",
-          :tree_state                => true})
+        = render(:partial => "shared/tree", :locals => {:tree => @menu_roles_tree, :name => "menu_roles_tree"})
```

Comment:
For consistency sake replace `"menu_roles_tree"` with `@menu_roles_tree.name` :)